service-python/Dockerfile.local

```diff
@@ -1,16 +1,16 @@
-ARG PYTHON_VERSION=3.9
+ARG WKHTML_VERSION=3.10.6-0.12.6-small
```

Comment:
Should we rename this file to `Dockerfile.secrel` since it's only used by SecRel?
We'll have to add a reminder to keep the 2 Dockerfiles in sync. Is there a better solution -- maybe https://docs.docker.com/build/building/multi-stage/?

Reply:
Using the multi-stage process, now we only have 1 `Dockerfile`

Reply:
After looking at your solution, I came up with a simpler solution that relies on recent changes to SecRel (being able to run a `script: ./scripts/build-images.sh` (declared in `config.yml`) before `docker build`). I pushed the solution to this PR.